src/action/common/configure_init_service.rs

```diff
@@ -189,13 +189,31 @@ impl Action for ConfigureInitService {
                 .await
                 .map_err(Self::error)?;
 
+                let domain = "system";
+                let service = "org.nixos.nix-daemon";
```

Comment:
iirc the original issue also had problems with the `darwin-store` service. Maybe this should be a `let services = vec![]` instead?

Reply:
That service is done via the `bootstrap_launchctl_service` action!

Reply:
Right!